ci/sync-version.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Comment:
This should be Apache, right @vmax ?

Reply:
@lolski `LICENSE` at the repo top says it's GNU AGPL v3, [ref](https://github.com/graknlabs/build-tools/blob/master/LICENSE)

Reply:
You're right, let's go with this then.